🌐 Add Traditional Chinese translation for `docs/zh-hant/docs/environment-variables.md`

Vincy1230 opened the pull request:
Add Traditional Chinese translation for docs/zh-hant/docs/environment-variables.md
為 environment-variables.md (環境變數) 章節添加正體中文翻譯

Vincy1230 committed:
🌐 Add Traditional Chinese translation for docs/zh-hant/docs/environment-variables.md

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+## 建立和使用環境變數
+
+你在**shell（終端機）**中就可以**建立**和使用環境變數，並不需要用到 Python：
```
```suggestion
你在 **shell（終端機）**中就可以**建立**和使用環境變數，並不需要用到 Python：
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+/// tip
+
+第二個參數是<a href="https://docs.python.org/zh-tw/3.8/library/os.html#os.getenv" class="external-link" target="_blank">`os.getenv()`</a>的預設返回值。
```
```suggestion
第二個參數是 <a href="https://docs.python.org/zh-tw/3.8/library/os.html#os.getenv" class="external-link" target="_blank">`os.getenv()`</a> 的預設回傳值。
```
「返回」是簡中用語，「回傳」是繁中，可以參考[這裡](https://zh.wikibooks.org/zh-tw/%E5%A4%A7%E9%99%86%E5%8F%B0%E6%B9%BE%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7%E8%A1%A8)。

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+////
+
+由於環境變數可以在程式碼之外設定、被程式碼讀取，並且不必與其他檔案一起儲存（提交到 `git`），因此通常用於配置或**設定**。
```
```suggestion
由於環境變數可以在程式碼之外設定，但可以被程式碼讀取，並且不必與其他檔案一起儲存（提交到 `git`），因此通常用於配置或**設定**。
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+你還可以為**特定的程式呼叫**建立特定的環境變數，該環境變數僅對該程式可用，且僅在其執行期間有效。
+
+要實現這一點，只需在同一行內、程式本身之前建立它：
```
```suggestion
要實現這一點，可以在程式執行前的同一行直接建立環境變數：
```
這樣會不會比較順？

Vincy1230 replied:
這麼做我擔心存在更改原文含義的嫌疑：

> To do that, create it right before the program itself, on the same line:

codingjenny replied:
```suggestion
要實現這一點，只需在同一行內（程式本身之前）建立它：
```
理解，那請問是否能修改成這樣？因為「頓號、」通常是在前後句子性質相同才會使用。

Vincy1230 replied:
你是正確的，但是這麼做必要性不高。無論哪一種譯文都不會影響句子的理解。原文中本就使用了並列的句式，在沒有明顯謬誤的情況下我們沒有必要在翻譯時修改原作者的構思邏輯。
稍後我会併入你的修改。

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+// 在這個程式呼叫的同一行中建立一個名為 MY_NAME 的環境變數
+$ MY_NAME="Wade Wilson" python main.py
+
+// 現在就可以讀到取環境變數了
```
```suggestion
// 現在就可以讀取到環境變數了
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+Hello Wade Wilson from Python
+
+// 在此之後環境這個變數將不再存在
```
```suggestion
// 在此之後這個環境變數將不再存在
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+/// tip
+
+你可以在<a href="https://12factor.net/zh_cn/config" class="external-link" target="_blank">The Twelve-Factor App: 配置</a>中了解更多資訊。
```
```suggestion
你可以在 <a href="https://12factor.net/zh_cn/config" class="external-link" target="_blank">The Twelve-Factor App: 配置</a>中了解更多資訊。
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+///
+
+## 類型和驗證
```
```suggestion
## 型別和驗證
```
「類型」是簡中用法，「型別」是繁中用法。

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+這些環境變數只能處理**文字字串**，因為它們是位於 Python 範疇之外的，必須與其他程式和作業系統的其餘部分相容（甚至與不同的作業系統相容，如 Linux、Windows、macOS）。
+
+這意味著從環境變數中讀取的**任何值**在 Python 中都將是一個 `str`，任何類型轉換或驗證都必須在程式碼中完成。
```
```suggestion
這意味著從環境變數中讀取的**任何值**在 Python 中都將是一個 `str`，任何型別轉換或驗證都必須在程式碼中完成。
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+假設你安裝了 Python，並將其安裝在目錄 `/opt/custompython/bin` 中。
+
+如果你選擇更新 `PATH` 環境變數，那麼安裝程式會將 `/opt/custompython/bin` 添加到 `PATH` 環境變數中。
```
```suggestion
如果你選擇更新 `PATH` 環境變數，那麼安裝程式會將 `/opt/custompython/bin` 加入到 `PATH` 環境變數中。
```
[這裡](https://zh.wikibooks.org/zh-tw/%E5%A4%A7%E9%99%86%E5%8F%B0%E6%B9%BE%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7%E8%A1%A8)顯示「添加」是簡中用法。

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+假設你安裝了 Python，並將其安裝在目錄 `C:\opt\custompython\bin` 中。
+
+如果你選擇更新 `PATH` 環境變數（在 Python 安裝程式中，這個選項是名為 `Add Python x.xx to PATH` 的勾選框——譯者註），那麼安裝程式會將 `C:\opt\custompython\bin` 添加到 `PATH` 環境變數中。
```
```suggestion
如果你選擇更新 `PATH` 環境變數（在 Python 安裝程式中，這個選項是名為 `Add Python x.xx to PATH` 的勾選框——譯者註），那麼安裝程式會將 `C:\opt\custompython\bin` 加入到 `PATH` 環境變數中。
```

codingjenny commented on docs/zh-hant/docs/environment-variables.md:
```diff
@@ -0,0 +1,298 @@
+
+環境變數（也稱為「**env var**」）是一個獨立於 Python 程式碼**之外**的變數，它存在於**作業系統**中，可以被你的 Python 程式碼（或其他程式）讀取。
+
+環境變數對於處理應用程式**設定**、作為 Python **安裝**的一部分等方面非常有用。
```
```suggestion
環境變數對於處理應用程式**設定**（例如作為 Python **安裝**的一部分等）非常有用。
```
這樣會不會比較順？

Vincy1230 replied:
這麼做我擔心存在更改原文含義的嫌疑：

> Environment variables could be useful for handling application settings, as part of the installation of Python, etc.

codingjenny replied:
```suggestion
環境變數對於處理應用程式**設定**（作為 Python **安裝**的一部分等方面）非常有用。
```
理解，那是否能修改成這樣？

Vincy1230 replied:
好的。同上。

Vincy1230 committed:
🔧 Apply suggestions from code review

Thank you @codingJenny for your suggestion. Already merged.
感谢 @codingjenny 提出的建议。已经并入。

Co-authored-by: YungYueh ChanLee <polo80456@gmail.com>

Vincy1230 committed:
🔧 Apply suggestions from code review

Thank you @codingjenny for your suggestion. Already merged.
感謝 @codingjenny 提出的建議。已經併入。

Co-authored-by: YungYueh ChanLee <polo80456@gmail.com>

yychanlee reviewed (approved):
LGTM

alejsdev commented:
Thank you for the help! @Vincy1230 :raised_hands: 
And thanks for the review @codingjenny :nerd_face:
